packages/backend/src/open_payments/auth/middleware.test.ts

```diff
@@ -4,21 +4,25 @@ import { URL } from 'url'
 import { v4 as uuid } from 'uuid'
 
 import { createAuthMiddleware } from './middleware'
-import { Grant, GrantJSON, AccessType, AccessAction } from './grant'
+import { GrantJSON, AccessType, AccessAction } from './grant'
 import { Config } from '../../config/app'
 import { IocContract } from '@adonisjs/fold'
 import { initIocContainer } from '../../'
-import { AppServices, PaymentPointerContext } from '../../app'
+import { AppServices } from '../../app'
+import { Body, RequestMethod } from 'node-mocks-http'
 import { HttpMethod, ValidateFunction } from 'openapi'
-import { setup } from '../../shared/routes.test'
+import { setup, SetupOptions } from '../../shared/routes.test'
```

Comment:
relocated in https://github.com/interledger/rafiki/commit/02920fd72c312dd457a9bd9f50b12c54954b0646
```suggestion
import { setup, SetupOptions } from '../payment_pointer/model.test'
```